start/blobstore.md

```diff
@@ -392,9 +392,10 @@ an object from a container that does not exist is a state problem, and should th
 <!--update to use ByteSource -->
 
 ### Uploading 
-As long as you use either `InputStream` or `File` as the payload for your blob, you should
+As long as you use either `ByteSource` or `File` as the payload for your blob, you should
 be fine. Note that in S3, you must calculate the length ahead of time, since it doesn't support
 chunked encoding.
+It is usually better to use a repeatable Payload like ByteSource instead of InputStream, since this allows parallel uploads and retrying on errors.
```

Comment:
Could you annotate as `ByteSource` and `InputStream`?

Reply:
Sorry, I dont follow? Annotate what?

Reply:
Got it, thx :) 
Commit amended.